src/query/expression/src/evaluator.rs

```diff
@@ -250,11 +250,14 @@ impl<'a> Evaluator<'a> {
             }
 
             (Scalar::Timestamp(ts), DataType::String) => Ok(Scalar::String(
```

Comment:
There is `run_cast_column` and `run_cast_scalar` in `evaluator.rs`, should we remove these?

Reply:
Seems need to remove these codes, or won't get into `to_xxx(date|timestamp)` functions.

Reply:
I'll do it in the next PR.